routes/notification.router.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const notificationService = require('../services/notification.service');
+
+const router = express.Router();
+
+router.get(':/userId', async (req, res) => {
+  const { userId } = req.params;
```

Comment:
You should remove userId from this route (make it just `/`) and get the id from the auth system. You can see an example [here](https://github.com/ctc-uci/oc-habitats-backend/blob/dev/routes/user.router.js#L27)